src/main/java/com/amazon/ion/impl/IonReaderTextRawTokensX.java

```diff
@@ -640,6 +641,16 @@ public final int nextToken() throws IOException
             return next_token_finish(IonTokenConstsX.TOKEN_COMMA, false);
         case '.':
             c2 = read_char();
+            if (c2 == '.') {
```

Comment:
Can you add a short comment explaining what's going on here?

Reply:
Sure.